### DISCUSSION TITLE
Using StatefulRedisPubSubConnection to publish, is there a way to filter specific consumers to publish the message to?

### DISCUSSION THREAD

**Discussion by dmdum** [redis/lettuce#2233] [Q&A] [answered]
Currently I have:

```
StatefulRedisPubSubConnection<String, String> pub;
. . .
pub.sync().publish(key, message);
```

that sends the message to the subscribers. 
Would it be possible to apply a filter to this so that the message will be sent to one specific subscriber?

**Comment by mp911de** [comment, marked as answer]
No, because Redis doesn't support this from the sender side.

**Reply by dmdum** [reply]
I see, thank you
